### PR TITLE
CoCC/SRC change to election eligibility

### PR DESCRIPTION
**Pull request by jberkus**
Per discussion in [Steering 226](https://github.com/kubernetes/steering/issues/226), this is one of two possible PRs, which automatically includes members of the Code of Conduct Committee and Security Response Committee as eligible to vote, regardless of number of contributions.

This is an exclusive alternative to PR #6757, for decision by the Steering Committee between the alternatives.

/sig contributor-experience
/area elections

### CODE REVIEW

**Comment by jberkus** [comment]
/hold
for steering discussion

File: elections/steering/2022/README.md
Comment (tpepper) [inline elections/steering/2022/README.md:83]: "Members of..." is potentially ambiguous here.  I think it should be more explicit like "Individuals listed as leadership chairs in sigs.yaml..."

Comment (micahhausler) [reply elections/steering/2022/README.md:83]: SRC has associates too, it would be good make clear if they are included

Comment (jberkus) [reply elections/steering/2022/README.md:83]: I think you're commenting on the wrong PR.  There are no leadership chairs here.

This one is just CoCC and SRC.  The listing for those two committees in SIGs.yaml is not in any way ambiguous.  CoCC has 5 members and SRC has 7, and nobody else is listed.  I can see adding "as listed in sigs.yaml" as a good confirmer.

Comment (micahhausler) [reply elections/steering/2022/README.md:83]: > I think you're commenting on the wrong PR. There are no leadership chairs here.

The SRC has [7 full committee members and 5 associate members](https://github.com/kubernetes/committee-security-response#security-response-committee-src), I was just asking for clarification if they are included.

Comment (liggitt) [reply elections/steering/2022/README.md:83]: > I can see adding "as listed in sigs.yaml" as a good confirmer.

With that made explicit, this PR looks good to me

Comment (jberkus) [reply elections/steering/2022/README.md:83]: Micah: what do the associate members do?  I'm wondering if they should be included, or not.

Comment (liggitt) [reply elections/steering/2022/README.md:83]: Associate responsibilities are described in https://github.com/kubernetes/committee-security-response/blob/main/security-release-process.md#associate, and are oriented towards public / non-disclosure work, which (in my opinion) is more visible and more likely to be covered via the normal devstats mechanism.

Comment (jberkus) [reply elections/steering/2022/README.md:83]: Is it possible to be an Associate and do very little/nothing all year?

Comment (liggitt) [reply elections/steering/2022/README.md]: I'll defer to Micah, but I suspect it's possible... if that's the case, maybe low devstats activity is an accurate reflection of little/no involvement, and collapsing to the normal threshold for contributor activity / voting eligibility seems ok to me

**Comment by jberkus** [comment]
Pushed edits to make the link to sigs.yaml clear, and to exclude Associate members.

**Comment by liggitt** [comment]
lgtm (steering)

File: elections/steering/2022/README.md
Comment (jberkus) [reply elections/steering/2022/README.md]: or, at least, it suggests that Associate Members should still need to file exceptions.

Comment (liggitt) [reply elections/steering/2022/README.md]: > or, at least, it suggests that Associate Members should still need to file exceptions.

right

**Comment by cblecker** [comment]
lgtm (also with my steering hat on)

**Comment by dims** [comment]
lgtm (also with my steering hat on)

**Review by mrbobbytables — COMMENTED** [review]
lgtm (steering hat)

**Review by justaugustus — APPROVED** [review]
/lgtm
/approve (Steering)

**Pull request merged**
